pandas/tests/extension/test_arrow.py

```diff
@@ -2145,31 +2265,12 @@ def test_str_rsplit():
     tm.assert_frame_equal(result, expected)
 
 
-@pytest.mark.parametrize(
-    "method, args",
-    [
-        ["partition", ("abc", False)],
-        ["rpartition", ("abc", False)],
-        ["removeprefix", ("abc",)],
-        ["casefold", ()],
-        ["encode", ("abc",)],
-        ["extract", (r"[ab](\d)",)],
-        ["findall", ("abc",)],
-        ["get_dummies", ()],
-        ["index", ("abc",)],
-        ["rindex", ("abc",)],
-        ["normalize", ("abc",)],
-        ["rfind", ("abc",)],
-        ["translate", ("abc",)],
-        ["wrap", ("abc",)],
-    ],
-)
-def test_str_unsupported_methods(method, args):
+def test_str_unsupported_extract():
     ser = pd.Series(["abc", None], dtype=ArrowDtype(pa.string()))
     with pytest.raises(
-        NotImplementedError, match=f"str.{method} not supported with pd.ArrowDtype"
+        NotImplementedError, match="str.extract not supported with pd.ArrowDtype"
```

Comment:
out of curiosity why is this one left out?

Reply:
str.extract was just a lot trickier to implement. I can try revisiting it in a followup